lib/package.js

```diff
@@ -140,66 +140,89 @@ exports.lookup = function(pkg) {
 };
 
 exports.getVersionMatch = getVersionMatch;
-function getVersionMatch(pkgVersion, versions) {
+function getVersionMatch(pkgVersion, versions, options) {
   // unescape pkgVersion for comparison
   if (pkgVersion)
     pkgVersion = decodeURIComponent(pkgVersion);
 
-  var latestPre;
-  var versionList = [];
-  var exactVersions = [];
+  var version;
+  var stableVersions = [];
+  var unstableVersions = [];
+  var exactStableVersions = [];
+  var exactUnstableVersions = [];
```

Comment:
For clarity, would it make sense to call these - `stableSemvers`, `unstableSemvers`, `stableTags`, `unstableTags`?